src/Wpf/Prism.Wpf/Modularity/IModuleCatalogExtensions.cs

```diff
@@ -20,12 +23,12 @@ public static IModuleCatalog AddModule<T>(this IModuleCatalog catalog, Initializ
         }
 
         /// <summary>
-        /// Adds the module.
+        /// Adds the module to the <see cref="IModuleCatalog"/>.
         /// </summary>
-        /// <param name="catalog">Catalog.</param>
-        /// <param name="name">Name.</param>
-        /// <param name="mode"><see cref="IModule"/>.</param>
-        /// <param name="dependsOn">Collection of module names (<see cref="ModuleInfo.ModuleName"/>) of the modules on which the module to be added logically depends on.</param>
+        /// <param name="catalog">The catalog to add the module to.</param>
+        /// <param name="name">Name of the module to be added.</param>
+        /// <param name="mode">Stage on which the module to be added will be initialized.</param>
```

Comment:
the updated mode description is really incorrect. It would be the `<see cref="InitializationMode"/>`